packages/flutter/test/material/dialog_test.dart

```diff
@@ -1319,54 +1319,6 @@ void main() {
     );
   });
 
-  testWidgets('AlertDialog widget contains route semantics from title for iOS', (WidgetTester tester) async {
```

Comment:
How come the descriptions says this uses an existing test, but this existing test is being removed?

Reply:
This is  testing a wrong behavior